Packs/Akamai_WAF/Integrations/Akamai_WAF/Akamai_WAF.yml

```diff
@@ -348,13 +252,692 @@ script:
+    - description: Phone of Admin Contact
+      name: adminContact_phone
+      required: true
+      type: textArea
+    - description: Fistname of Tech Contact
+      name: techContact_firstName
+      required: true
+      type: textArea
+    - description: Lastname of Tech Contact
+      name: techContact_lastName
+      required: true
+      type: textArea
+    - description: Email of Tech Contact
+      name: techContact_email
+      required: true
+      type: textArea
+    - description: Phone of Tech Contact
+      name: techContact_phone
+      required: true
+      type: textArea
+    - description: Orgnization name
+      name: org_name
+      required: true
+      type: textArea
+    - description: Orgnization Country
+      name: org_country
+      required: true
+      type: textArea
+    - description: Orgnization City
+      name: org_city
+      required: true
+      type: textArea
+    - description: Orgnization Region
+      name: org_region
+      required: true
+      type: textArea
+    - description: Orgnization PostalCode
+      name: org_postalCode
+      required: true
+      type: textArea
+    - description: Orgnization Phone
+      name: org_phone
+      required: true
+      type: textArea
+    - description: Orgnization Address
+      name: org_addressLineOne
+      required: true
+      type: textArea
+    description: Create a new enrollment
+    name: akamai-create-enrollment
+    outputs:
+    - contextPath: Akamai.Create.NewGroup.changes.[0]
+      description: Enrollment_path
+      type: string
+  - arguments:
+    - name: contractId
+      required: true
+    description: List enrollments of a specific contract
+    name: akamai-list-enrollments
+  - arguments:
+    - description: domainName
+      name: domainName
+      required: true
+      type: textArea
+    - description: Group Id
+      name: groupId
+      required: true
+    description: Create a domain with properties and DC
+    name: akamai-create-domain
+  - arguments:
+    - description: The domain Name that the new property is added to
+      name: domainName
+      required: true
+    - description: New property name
+      name: property_name
+      required: true
+    - description: property type
+      name: property_type
+      required: true
+    - description: '"CNAME" or "A"'
+      name: static_type
+    - description: static_server
+      name: static_server
+    - description: server_1
+      name: server_1
+    - description: server_2
+      name: server_2
+    - description: weight_1
+      name: weight_1
+    - description: weight_2
+      name: weight_2
+    description: Update a property for a specific domain
+    name: akamai-update-property
+  - arguments:
+    - name: enrollment_path
+      required: true
+    - defaultValue: third-party-csr
+      name: allowedInputTypeParam
+    description: Get the cps code
+    name: akamai-get-change
+    outputs:
+    - contextPath: Akamai.get_change.csrs.[0].csr
```

Comment:
change to Akamai.{ObjectName} --> ObjectName should be what the commands actually does.